Entities/ThrownDevice.h

```diff
@@ -133,6 +133,7 @@ namespace RTE {
 		float m_MaxThrowVel; //!< The maximum throw velocity this gets when thrown.
 		long m_TriggerDelay; //!< Time in millisecs from the time of being thrown to triggering whatever it is that this ThrownDevice does.
 		bool m_ActivatesWhenReleased; //!< Whether this activates when its throw is started, or waits until it is released from the arm that is throwing it.
+		const MovableObject *m_StrikerLever; //!< Striker lever particle MovableObject preset instance.
```

Comment:
Do you **want** this to be an MO? If it's supposed to be a particle, why not limit to an MOSprite or MOSParticle or whatever you actually want it to be?

Reply:
Yeah, what do you mean? Shell casing on a Round can be MOSR too, why shouldn't that be the case for grenades as well?

Reply:
Yeah, I guess Shell casing uses MovableObject as well. What I meant by this is that there're benefits to having maximal specificity in your class use. Data used MO everywhere and that's why you always have to ToWhatever in lua, or cast things around in cpp. Here, for example, I wonder if we could use MOSprite instead - that'd cover MOSParticle and MOSRotating subclasses.

The advantage of just using MO here is that people will never have trouble figuring out what object type to put in there, but keeping it more specific where possible is helpful imo.

Reply:
But what if you want it to be an Actor??? :p

Reply:
I mean, Actor inherits from MOSprite so you could still do that. You wouldn't be able to have it be an MOPixel though, which is the main loss I guess. Also, of course you could specify it more and require it to be an MOSParticle or an MOSRotating or something, but I don't think that's needed.

Anyway, think about what would be best for it I guess. It's not a huge deal either way, but since you're putting this into place and you do most of the ini and lua stuff, you're probably the best person to make the call here.